README.md

```diff
@@ -51,12 +51,17 @@ the future.
 
 ## Documentation
 
-This is the main project repository. In this repository, documentation
-regarding the [design](docs/internal/design),
-[requirements](docs/internal/requirements) and
-[specifications](docs/public/specifications) for the Zonemaster implementation
-are available.
-We also have a brief [user guide](USING.md).
+This is the main project repository. In this repository, most documentation of
+Zonemaster is found.
+
+In the [public documentation](docs/public/SUMMARY.md) you will e.g. find
```

Comment:
Did you consider providing a README file at docs/public/README.md to refer to instead?

Reply:
Do you mean renaming SUMMARY.md to README.md? That would be fine to me.

Reply:
I was thinking about an extra README.md file that could explains what can be found in the public documentation.

Reply:
Or renaming the SUMMARY.md to README.md and adding an introduction at the top?

Reply:
The SUMMARY.md file is required to build the HTML documentation:

> The summary file is used by mdBook to know what chapters to include, in what order they should appear, what their hierarchy is and where the source files are. Without this file, there is no book.
>
> This markdown file must be named SUMMARY.md. Its formatting is very strict and must follow the structure outlined below to allow for easy parsing. Any element not specified below, be it formatting or textual, is likely to be ignored at best, or may cause an error when attempting to build the book.
>
>https://rust-lang.github.io/mdBook/format/summary.html

Reply:
What is your take on this?

Reply:
Yes, I created a README.md file on your (@pnax) suggestion, and changed so that main README.md links to that. In the new README.md there is a link to SUMMARY.md.

Reply:
My bad, I didn't see it. Thanks for the pointer.

Reply:
(My comments were not published.)